config/livewire.php

```diff
@@ -0,0 +1,113 @@
+<?php
```

Comment:
Could you explain how this file was changed? Is this an additional package you need?

Reply:
It was an additional package I needed.